### ISSUE TITLE
Frontend: Browser-sync

### ISSUE THREAD

**Issue by michaelpande** [olehermanse/nowcode#27]
We should support browser-sync for automatically updating the GUI while working with it.

**Comment by olehermanse** [comment]
@michaelpande Does this require anything from the backend?

**Comment by michaelpande** [comment]
No, it's fairly simple to set up. I'll see what I can do later this week.
